### ISSUE TITLE
Connect SdK can not support video type of ts

### ISSUE THREAD

**Issue by qs991011** [ConnectSDK/connectsdk.com#10]
When I transfer TS files to TV through connect SDK, there will be an error message
`
 mMediaPlayer.load(mApiClient, mediaInformation, true).setResultCallback(new ResultCallback<MediaChannelResult>() {

                @Override
                public void onResult(MediaChannelResult result) {
                    Status status = result.getStatus();

                    if (status.isSuccess()) {
                        webAppSession.mLaunchSession.setSessionType(LaunchSessionType.Media);
                        mMediaPlayer.setActiveMediaTracks(mApiClient, new long[]{MEDIA_TRACK_ID});
                        Util.postSuccess(listener, new MediaLaunchObject(webAppSession.mLaunchSession, CastService.this));
                    } else {
                        Util.postError(listener, new ServiceCommandError(status.getStatusCode(), status.getStatusMessage(), status));
                    }
                }
            });
`

the result of status is not success. so I would like to know if it is not supported for TS files。Thank you.

**Comment by yjojovasuka** [comment]
yes you are right , you have to convert ts to mp4 (you can use ffmpeg) and than you can use it.
